gradle/dependencies.gradle

```diff
@@ -66,7 +66,8 @@ def support = [
         lifecycle: [
             common: "android.arch.lifecycle:common:${versions.archComponents}",
             compiler: "android.arch.lifecycle:compiler:${versions.archComponents}",
-            extensions: "android.arch.lifecycle:extensions:${versions.archComponents}"
+            extensions: "android.arch.lifecycle:extensions:${versions.archComponents}",
+            runtime: "android.arch.lifecycle:runtime:1.0.0"
```

Comment:
Do runtime have different release cycle, so you don't use the same version?

Reply:
Yeah runtime is already stable and released